gimme_aws_creds/okta.py

```diff
@@ -112,7 +116,6 @@ def stepup_auth(self, embed_link, state_token=None):
         flow_state = self._get_initial_flow_state(embed_link, state_token)
 
         while flow_state.get('apiResponse', {}).get('status') != 'SUCCESS':
-            time.sleep(0.5)
```

Comment:
These sleeps were added to prevent hammering the API endpoints if we didn't get a success message back.  Unless there is a particular reason this were removed, lets add this back in.

Reply:
Not sure why it got removed but it is now back.